packages/edit-post/src/components/editor-regions/index.js

```diff
@@ -31,7 +31,9 @@ function EditorRegions( { footer, header, sidebar, content, publish, className }
 					aria-label={ __( 'Editor content' ) }
 					tabIndex="-1"
 				>
-					{ content }
+					<Popover.Slot>
```

Comment:
What's the reasoning for including this in `EditorRegions`? Should we remove the existing PopoverSlot included in the Layout?